src/vs/base/parts/quickinput/browser/quickInput.ts

```diff
@@ -740,6 +741,14 @@ class QuickPick<T extends IQuickPickItem> extends QuickInput implements IQuickPi
 		this.update();
 	}
 
+	set additionalToggles(toggles: IQuickInputToggle[] | undefined) {
+		// HACK: Filter out toggles here that are not concrete Toggle objects. This is to workaround
```

Comment:
I think it's about time I move qp into platform/*/browser....

Anyway I'm fine with this.

Reply:
If you do this, going the `IBrowserQuickPick extends IQuickPick` route is probably easiest/safest. I expect you'll hit a bunch of problems if you try move `IQuickPick` to browser